geniza/corpus/tests/test_corpus_models.py

```diff
@@ -437,6 +437,14 @@ def test_str(self):
         unsaved_doc = Document()
         assert str(unsaved_doc) == "?? (PGPID ??)"
 
+    def test_historical_date_display(self):
+        """Should display the historical document date with its calendar name"""
+        doc = Document.objects.create(doc_date_original="507", doc_date_calendar="h")
```

Comment:
please use the code defined on the class instead of a bare letter

```suggestion
        doc = Document.objects.create(doc_date_original="507", doc_date_calendar=Document.CALENDAR_HIJRI)
```